### DISCUSSION TITLE
Nominate @fg-j to the Stacks Contributor team

### DISCUSSION THREAD

**Discussion by robdimsdale** [paketo-buildpacks/feedback#157] [Stacks Team]
@fg-j has been collaborating extensively with me on the jammy-tiny stacks work. I would like to nominate them to the contributor team to make it easier for them to contribute to stacks in the future.

@paketo-buildpacks/stacks-maintainers 👍 / 👎

**Comment by robdimsdale** [comment]
@paketo-buildpacks/steering-committee looks like we have a majority of stacks maintainers in favor of @fg-j becoming a stacks contributor.

**Comment by ryanmoran** [comment]
Congrats @fg-j on becoming a Stacks Contributor!! :tada:
